src/__tmpdata/pages.json

```diff
@@ -1,23 +1,37 @@
 {
   "meta": {},
+  "themepage":{},
```

Comment:
why do we want to add `themepage` & `topicpage` to this file instead of creating new files for them?

Reply:
Oh we can do it either way. I'd just been putting all the content in pages.json that really should live in the CMS. You'll see json in there for the service, services, dept page.